src/annotator/adder.js

```diff
@@ -51,6 +51,23 @@ function nearestPositionedAncestor(el) {
   return parentEl;
 }
 
+/**
+ * Recursively find the highest z-index for this element and its parents
+ *
+ * @param {Element|null|undefined } element - element that has the current selection
```

Comment:
Is the `undefined` needed here? It looks like this function will always be passed an element or `null`.

Reply:
Yes it is needed because `focusNode.parentElement` could return `undefined`.
The `focusNode` on a `Selection` objection, in my experience returns a text node.